intents/oversights/looking_at_tails.py

```diff
@@ -0,0 +1,175 @@
+"""
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+"""This module detects the oversight 'Looking at tails to find causes'
+when the top-k oversight is called.
+
+This oversight occurs when the user sorts according a metric and some other
+metric which he has requested for remaing same for the entire list as it is
+in the top-k entries. Here the user might get misleaded by seeing the other
+metric - he might make wrong view that only in the top-k this metric looks
+like this. But actually the metric is closeby for the entire list.
+
+Todo(when the date format is updated) - add the test for date columns
+"""
+import sys
```

Comment:
We should only add this for tests, right?

Reply:
yes, removed it.